### ISSUE TITLE
Browser Refresh not working in warning state

### ISSUE THREAD

**Issue by eastcoastcoder** [devloco/create-react-wptheme#59]
This could be an issue on my end but I can't seem to get handleWarnings to fire which is breaking browser refresh in a warning state. That said I can just run my client like normal with react-scripts but I was hoping to continue running with wpstart.

Here's the block of code that catches. (Inside wpThemeClient.js)

![image](https://user-images.githubusercontent.com/4449468/88181068-523abe00-cbfc-11ea-8d11-01cd1afecbcd.png)

**Comment by michaelsoriano** [comment]
I'm having a similar issue. Any luck in a solution?

**Comment by eastcoastcoder** [comment]
Not really, however the build command works fine with warnings. So I just use the normal react scripts start command for development and the wpscripts build command when I'm ready to publish.
